aiida/orm/nodes/node.py

```diff
@@ -1024,18 +1025,10 @@ def store(self, with_transaction=True, use_cache=None):  # pylint: disable=argum
                 self._store(with_transaction=with_transaction, clean=True)
 
             # Set up autogrouping used by verdi run
-            from aiida.orm.autogroup import current_autogroup, Autogroup, VERDIAUTOGROUP_TYPE
-            from aiida.orm import Group
-
-            if current_autogroup is not None:
-                if not isinstance(current_autogroup, Autogroup):
-                    raise exceptions.ValidationError('`current_autogroup` is not of type `Autogroup`')
-
```

Comment:
No, we manage internally the class, so this validation is just going to slow down storing (this is the performance improvement I was mentioning).
Anyway if the class is wrong because the user messed up, then the code would raise elsewhere. I'll keep it removed.